src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1096,5 +1102,36 @@ private static void ThrowMinMaxException<T>(T min, T max)
         {
             throw new ArgumentException(SR.Format(SR.Argument_MinMaxValue, min, max));
         }
+
+        public static double ScaleB(double f, int n)
```

Comment:
nit: use the [documented](https://docs.microsoft.com/en-us/dotnet/api/system.math.scaleb?view=netcore-3.1) parameter names:
```suggestion
        public static double ScaleB(double x, int n)
```